frontend/checks-runner.ts

```diff
@@ -0,0 +1,160 @@
+import checkbox, { Separator } from '@inquirer/checkbox';
+import chalk from 'chalk';
+import { spawn } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+
+// Define the structure of package.json
```

Comment:
Is this comment necessary? I'm not sure it provides any value.

Reply:
Code has been refactored.